problems/finger_pointing_and_blaming.md

```diff
@@ -0,0 +1,48 @@
+---
+slug: finger-pointing-and-blaming
+stages:
+  - research
+short_description: Finger-Pointing and Blaming is the act of blaming someone for a problem instead of trying to fix or take accountability. This results in a toxic work culture where nobody feels safe and there's no trust.
+tags:
+  - finger-pointing
+  - blaming
+  - culture
+  - mentality
+  - trust
+keywords:
+  - team player
+  - problem
+  - satisfaction
+  - responsibilities
+  - leadership
+  - help
+  - performance
+  - environment
+  - efficiency
+  - empowerment
+---
+
+# Finger-Pointing And Blaming
+
+**TL;DR**
+
+Finger-Pointing and Blaming is the act of blaming someone for a problem instead of trying to fix or take accountability. This results in a toxic work culture where nobody feels safe and there's no trust. It is important to combat this problem quickly in order to establish a healthy and a more efficient work environment. Read more to find more about this!
+
+## Impact Of The Blame Culture
+
```

Comment:
Please, add an image to your article. It should be an article which license allows us to use it.
Use similar syntax:
![Finger](/files/finger.png)
[Finger](URL source)